streams/src/main/java/org/apache/kafka/streams/StreamsMetrics.java

```diff
@@ -170,29 +181,35 @@ void recordThroughput(final Sensor sensor,
     /**
      * Generic method to create a sensor.
      * Note that for most cases it is advisable to use
-     * {@link #addThroughputSensor(String, String, String, Sensor.RecordingLevel, String...)}
-     * or {@link #addLatencyAndThroughputSensor(String, String, String, Sensor.RecordingLevel, String...)} to ensure
-     * metric name well-formedness and conformity with the rest of the streams code base.
+     * {@link #addRateTotalSensor(String, String, String, RecordingLevel, String...) addRateTotalSensor()}
+     * or {@link #addLatencyRateTotalSensor(String, String, String, RecordingLevel, String...) addLatencyRateTotalSensor()}
+     * to ensure metric name well-formedness and conformity with the rest of the Kafka Streams code base.
      * However, if the above two methods are not sufficient, this method can also be used.
      *
      * @param name           name of the sensor.
      * @param recordingLevel the recording level (e.g., INFO or DEBUG) for this sensor
      * @return The added sensor.
+     * @see #addRateTotalSensor(String, String, String, RecordingLevel, String...)
+     * @see #addLatencyRateTotalSensor(String, String, String, RecordingLevel, String...)
+     * @see #removeSensor(Sensor)
      */
     Sensor addSensor(final String name,
                      final Sensor.RecordingLevel recordingLevel);
```

Comment:
Isn't this method subsumed by the method on line 214? Shall deprecate this method?

Reply:
Looks like we could deprecate it, but it's not listed for deprecation in the KIP.  I'll defer to @guozhangwang on this one.

Reply:
I think these are supposed to be two overloaded functions but yeah I agree that the overloading here seems not worth much :P I think it's okay just to leave as is.